app/js/hoc/withFooter.scss

```diff
@@ -0,0 +1,14 @@
+$footer-height: 58px;
+
+.container {
+  padding-bottom: $footer-height;
```

Comment:
It is a bit scary to change the container class, as we may use it in lot of places.
Do you think we could have a wrapper dedicated class?

Reply:
SO, I just changed the footer a bit so that it stays at the bottom withotu being fixed. It took way too long for what it does, but at least it works now. 

I still need the padding, because otherwise, the content goes under the footer.
However, it's not super scary, because it's bottom and also, it's just a padding, so if there is nothing under the container (like always I think), it will never be seen.